Cargo.toml

```diff
@@ -9,6 +9,9 @@ license = "MIT"
 categories = ["rust-patterns"]
 keywords = ["builder", "builder_pattern"]
 
+[features]
+better_error_message = []
```

Comment:
May I suggest a more concise name?
How about `better_error`? It kinda rhythms :)